src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset enabled class="forma">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            class="field"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="number"
+            class="field"
+            min="1" max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+            class="field"
+          >
+        </label>
+
+        <label>
+            I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="I accept the term of the agreement"
+            class="field"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+           E-mail:
+          <input
+            type="email" name="email"
```

Comment:
move every attribute to new line
```suggestion
            type="email"
            name="email"
```

Reply:
Fix everywhere please